manage-data/ingest/ingesting-data-from-applications/ingest-logs-from-nodejs-web-application-using-filebeat.md

```diff
@@ -20,514 +22,443 @@ This guide demonstrates how to ingest logs from a Node.js web application and de
 
-```txt
-# =============================== Elastic Cloud ================================
+::::
 
-# These settings simplify using Filebeat with the Elastic Cloud (https://cloud.elastic.co/).
+::::{step} Stop the Node.js script
 
-# The cloud.id setting overwrites the `output.elasticsearch.hosts` and
-# `setup.kibana.host` options.
-# You can find the `cloud.id` in the Elastic Cloud web UI.
-cloud.id: my-deployment:yTMtd5VzdKEuP2NwPbNsb3VkLtKzLmldJDcyMzUyNjBhZGP7MjQ4OTZiNTIxZTQyOPY2C2NeOGQwJGQ2YWQ4M5FhNjIyYjQ9ODZhYWNjKDdlX2Yz4ELhRYJ7 <1>
+After confirming that both `webserver.js` and `webrequests.js` run as expected, enter *CTRL + C* to stop the Node.js script.
 
-# The cloud.auth setting overwrites the `output.elasticsearch.username` and
-# `output.elasticsearch.password` settings. The format is `<user>:<pass>`.
-cloud.auth: elastic:591KhtuAgTP46by9C4EmhGuk <2>
-```
+::::
 
-1. Uncomment the `cloud.id` line and add the deployment’s Cloud ID. You can include or omit the *<deploymentname>:* prefix at the beginning of the Cloud ID. Both versions work fine. Find your Cloud ID by going to the {{kib}} main menu and selecting Management > Integrations, and then selecting View deployment details.
-2. Uncomment the `cloud.auth` line and add the username and password for your deployment that you recorded when you created your deployment. The format is *<username>:<password>*, for example *elastic:57ugj782kvkwmSKg8uVe*.
+:::::
 
 
-**Configure Filebeat inputs**
+## Get Filebeat
 
-Filebeat has several ways to collect logs. For this example, you’ll configure log collection manually.
+To collect and forward your Node.js application logs to Elastic Cloud, you'll need to set up Filebeat as your log shipper.
 
-In the *filebeat.inputs* section of *filebeat.yml*, set *enabled:* to *true*, and set *paths:* to the location of your web server log file. In this example, set the same directory where you saved *webserver.js*:
+:::::{stepper}
 
-```txt
-filebeat.inputs:
+::::{step} Download and install Filebeat
 
-# Each - is an input. Most options can be set at the input level, so
-# you can use different inputs for various configurations.
-# Below are the input specific configurations.
+[Download Filebeat](https://www.elastic.co/downloads/beats/filebeat) and unpack it on the local server from which you want to collect data.
 
-- type: log
+::::
 
-  # Change to true to enable this input configuration.
-  enabled: true
+::::{step} Configure Filebeat to access {{ech}} or {{ece}}
 
-  # Paths that should be crawled and fetched. Glob based paths.
-  paths:
-    - /path/to/logs/log.json
+In *<localpath>/filebeat-<version>/* (where *<localpath>* is the directory where Filebeat is installed and *<version>* is the Filebeat version number), open the `filebeat.yml` configuration file for editing.
```

Comment:
Consider formatting directory paths as inline code to signal to the reader that it's not prose. 

```suggestion
In `<localpath>/filebeat-<version>/` (where `<localpath>` is the directory where Filebeat is installed and `<version>` is the Filebeat version number), open the `filebeat.yml` configuration file for editing.
```